src/lib.rs

```diff
@@ -135,6 +135,10 @@ pub struct Diagnostic {
 
     /// The diagnostic's message.
     pub message: String,
+
+    /// A number used to associate children diagnostics with their
```

Comment:
Can you adding something about this being experimental/not part of the language protocol spec?